code/__defines/client.dm

```diff
@@ -1,2 +1,2 @@
-#define CLIENT_MIN_FPS 0
+#define CLIENT_MIN_FPS -1
```

Comment:
I don't know what this is doing. Please move all the client FPS stuff to a separate PR unless the system does not work without it.